src/kakarot/accounts/uninitialized_account.cairo

```diff
@@ -6,50 +6,38 @@ from starkware.cairo.common.cairo_builtins import HashBuiltin, BitwiseBuiltin
 from starkware.cairo.common.alloc import alloc
 from starkware.starknet.common.syscalls import get_caller_address, replace_class, library_call
 
-// We are intentionally causing a storage_slot collision here,
-// by defining these variables in both `uninitialized_account` and `account_contract`.
-// We are defining them here instead of in the account library, so as to not depend
-// on content of the account library in uninitialized_account and ensure a fixed class hash.
-@storage_var
-func Account_evm_address() -> (evm_address: felt) {
-}
-
-@storage_var
-func Account_kakarot_address() -> (kakarot_address: felt) {
+@contract_interface
+namespace IKakarot {
```

Comment:
we could as well define a `namespace IAccount` with the initialize, the use it instead of the raw library call